fw/t/unit/test_hpack.c

```diff
@@ -1519,401 +1519,6 @@ TEST(hpack, enc_table_index)
 #undef HDR_VALUE_3
 }
 
-TEST(hpack, enc_table_rbtree)
```

Comment:
IIUC this test just verifies correctness of our rb-tree implementation. What problems do we have with this test?

Reply:
RBtree structure (ie child/parent/left/right) nodes are depends on comparison function.